### DISCUSSION TITLE
Multiple Masks

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#472] [Ask A Question]
Can I use more than one poly masks (ignore 2 or more separate areas)? If so can someone provide an example?
I would like to ignore following 2 coordinates generated by image-map.net

`<map name="image-map">
    <area target="" alt="tree" title="tree" href="" coords="1703,165,1663,95,1649,57,1649,8,1839,6,1830,64,1752,144" shape="poly">
    <area target="" alt="branches" title="branches" href="" coords="1,671,6,1,184,5,370,16,421,14,373,77,500,46,468,130,602,128,513,215,428,236,292,316" shape="poly">
</map>`

**Comment by Trashmee** [comment]
Beta3 documentation vanished, so I'm not 100% certain. But as I recall multiple masks is a new feature of the (yet unreleased) beta4 code.
If you check the beta4 documentation it should be quite easy to configure the coords from your above sample. Just search for "mask:" in below link:

https://github.com/blakeblackshear/frigate/tree/beta4#Configuration

**Reply by bm55** [reply]
I tried following and no error so I think it's working in beta3

> mask: 
      - poly,1703,165,1663,95,1649,57,1649,8,1839,6,1830,64,1752,144
      - poly,1,671,6,1,184,5,370,16,421,14,373,77,500,46,468,130,602,128,513,215,428,236,292,316

**Reply by bm55** [reply]
Lookin at the mjpeg endpoint with mask flag, it looks like only the first poly coordinates are being used - I see the red motion squares in the 2nd mask area but not in the 1st mask area. Maybe on next version we will be able to use multiple poly masks @blakeblackshear ?

**Reply by bm55** [reply]
Or I am not using the poly mask correctly in config
